test/parallel/test-uv-binding-constant.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const uv = process.binding('uv');
+
+// Ensures that the `UV_...` values in process.binding('uv')
+// are constants.
+
+const keys = Object.keys(uv);
+keys.forEach((key) => {
+  if (key === 'errname')
+    return; // skip this
+  const val = uv[key];
+  assert.throws(() => uv[key] = 1,
```

Comment:
AFAICT assignment is not a valid lambda expression, you need to `() => { uv[key] = 1; }`